.env.test

```diff
@@ -4,7 +4,7 @@ FLASK_APP=todo_app/app
 
 # Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
 FLASK_DEBUG=true
-
+FLASK_ENV=docker
```

Comment:
Leaving this as development or similar might make more sense, since it's not _necessarily_ true that the tests will be run in Docker?